modelcheck/build.gradle.kts

```diff
@@ -5,24 +5,27 @@ buildscript {
 }
 
 plugins {
-    id("kotlin-conventions")
+    id("backend-conventions")
 }
 
 val mpsZip by configurations.creating
 
 dependencies {
-    compileOnly("log4j:log4j:1.2.17")
     mpsZip(libs.mps)
 
+    compileOnly("commons-logging:commons-logging:1.2")
+
     compileOnly(zipTree({ mpsZip.singleFile }).matching {
-        include("lib/mps-environment.jar")
-        include("lib/mps-openapi.jar")
         include("lib/mps-core.jar")
+        include("lib/mps-environment.jar")
         include("lib/mps-platform.jar")
         include("lib/mps-project-check.jar")
+        include("lib/mps-openapi.jar")
+        include("lib/mps-logging.jar")
         include("lib/platform-api.jar")
         include("lib/util.jar")
 
+        include("lib/mps-modelchecker.jar")
```

Comment:
Question: I can't find this JAR. Is this correct? Where does it come from?

Reply:
Strange... it must have been there at some point... or was it just a dream? I'll delete this.